### ISSUE TITLE
Why copying is done twice ?

### ISSUE THREAD

**Issue by storm999** [mifi/lossless-cut#1360]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

The program first creates a new file of each trimmed part and then merges them into bigger file. During this process, same data is copied twice.
Why do you need to re-copy for merging parts ? Can't you just merge files without copying for 2nd time.

**Comment by mifi** [comment]
I believe your question is the same as #963
tldr: I don't know how to do ffmpeg concat without first creating separate files. if anyone knows, i'm happy to learn about it
